config.py

```diff
@@ -16,6 +16,7 @@ class Config(object):
     REDIS_PORT = os.getenv('REDIS_PORT')
     REDIS_DB = os.getenv('REDIS_DB', 0)
     SECURE_COOKIES = strtobool(os.getenv('SECURE_COOKIES', 'True'))
+    EDIT_EVENT_DATES_ENABLED = os.getenv('EDIT_EVENT_DATES_ENABLED') == 'True'
```

Comment:
This could use `strtobool` like the line above but with a default value of `'false'`.